gateways/core/artifact/file/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM centos:7
```

Comment:
why are all these built from centos? can they be from scratch? do you need to ability to exec into the gateway pods?

Reply:
Good point, I only need the File Gateway to be built from centos so that user can debug pod to search file system. But other than that gateway, we can build all from scratch.